src/locations.jl

```diff
@@ -113,6 +113,7 @@ function determine_save_path(name, rel=nothing)::String
         0 == first(uv_access(path, W_OK))
     end
     if path_ind === nothing
+        @error "No writable path exists to save the data, make sure there exists as writable path in your DataDeps Load Path" ENV["DATADEPS_LOAD_PATH"]
```

Comment:
`ENV["DATADEPS_LOAD_PATH"]` does not contain the actual complete loadpath.
`cands` does though, as that is what `preferred_paths` returns.
It should probably say "The current load path contains:"
since it can be changed

Also I am not sure about the capitalisation.
Nor one the pros and cons of using `@error` vs including this in the error message.

Reply:
When I have error messages that are more than a couple of words (esp if I'm including calculated values), I just prefer the formatting of `@error`. Happy to put it all directly in the exception if you prefer.

Reply:
ok, lets try it your way.
Can change it later  if someone manages to get  the  error  but  not  the log  message